tests/Runtime-Microsoft.Android.Sdk/Microsoft.Android-Tests.projitems

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <_PackageName>Microsoft.Android_Tests</_PackageName>
+  </PropertyGroup>
+  <ItemGroup>
+    <TestApk Include="$(OutputPath)$(_PackageName)-Signed.apk">
+      <Package>$(_PackageName)</Package>
+      <InstrumentationType>xamarin.android.runtimetests.TestInstrumentation</InstrumentationType>
+      <ResultsPath>$(MSBuildThisFileDirectory)..\..\TestResult-$(_PackageName).xml</ResultsPath>
+      <TimingDefinitionsFilename>$(MSBuildThisFileDirectory)..\..\build-tools\scripts\TimingDefinitions.txt</TimingDefinitionsFilename>
+      <TimingResultsFilename>$(MSBuildThisFileDirectory)..\..\TestResult-$(_PackageName)-times.csv</TimingResultsFilename>
+      <ApkSizesInputFilename>apk-sizes-$(_PackageName)-$(Configuration)$(TestsFlavor).txt</ApkSizesInputFilename>
+      <ApkSizesDefinitionFilename>$(MSBuildThisFileDirectory)apk-sizes-definitions.txt</ApkSizesDefinitionFilename>
+      <ApkSizesResultsFilename>$(MSBuildThisFileDirectory)..\..\TestResult-$(_PackageName)-values-$(Configuration).csv</ApkSizesResultsFilename>
+    </TestApk>
+  </ItemGroup>
+
+  <ItemGroup>
+    <TestApkInstrumentation Include="xamarin.android.runtimetests.NUnitInstrumentation">
+      <Package>$(_PackageName)</Package>
+      <ResultsPath>$(OutputPath)TestResult-$(_PackageName).xml</ResultsPath>
+    </TestApkInstrumentation>
+
+    <TestApkPermission Include="READ_EXTERNAL_STORAGE">
```

Comment:
Do these permissions still work on API 30?

Reply:
I copied these over to follow the convention that the other test projects set, they don't seem to be causing any issues though

> Executing: /Users/runner/Library/Android/sdk/platform-tools/adb   shell pm grant Microsoft.Android_Tests android.permission.READ_EXTERNAL_STORAGE